### DISCUSSION TITLE
Github Copilot is not accessible anymore.

### DISCUSSION THREAD

**Discussion by samyogdhital** [community/community#9641] [Copilot Conversations] [answered]
Previously I had got access to GitHub copilot. But recently I forgot the password of my GitHub account and asked the GitHub support to completely reset the account and release the email for me. After sometime I made the account from same email on GitHub and now I am not able to use GitHub Copilot.. Can anyone help me with this ?

**Comment by davecheney** [comment, marked as answer]
I've reconfirmed your invite.
